src/Components/ErrorFallbackComponent/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { FallbackProps } from "react-error-boundary";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import Modal from "@mui/material/Modal";
+import { useState } from "react";
+import { useTranslation } from "react-i18next";
+
+const style = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  bgcolor: "background.paper",
+  border: "2px solid #fff",
+  boxShadow: 24,
+  p: 4,
+  maxWidth: "100%",
+  "@media (max-width: 400px)": {
+    width: "90%",
+  },
+};
```

Comment:
you should use the `satisfies` operator for object literals.

```js
import type { SxProps } from '@mui/material';
```

```suggestion
} satisfies SxProps;
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator